calculator_spec.rb

```diff
@@ -0,0 +1,48 @@
+require './calculator'
+
+# TESTES DE UNIDADE
+RSpec.describe Calculator do
+    context 'when calculates one of the four basic mathematics operations' do
+        it 'calculates sum' do
+            subject = described_class.new
+
+            result = subject.sum(2, 2)
+            expect(result).to eq 4
```

Comment:
Aqui a gente pode pular uma linha, assim a gente deixa separada a etapa de exercise da etapa de verify (https://thoughtbot.com/blog/four-phase-test).
```suggestion
            result = subject.sum(2, 2)

            expect(result).to eq 4
```

Reply:
Isso se aplica para os outros testes da classe.